kitsune/community/utils.py

```diff
@@ -3,40 +3,48 @@
 from operator import itemgetter
 
 from django.conf import settings
+from django.contrib.auth.models import Group
 from django.core.cache import cache
 from django.db.models import Count
 
 from kitsune.products.models import Product
-from kitsune.users.models import User, UserMappingType
+from kitsune.users.models import CONTRIBUTOR_GROUP, User, UserMappingType
 from kitsune.wiki.models import Revision
 
 from elasticsearch_dsl import A
 from kitsune.search.v2.documents import AnswerDocument, ProfileDocument
 
 
-def top_contributors_questions(start=None, end=None, locale=None, product=None, count=10):
+def top_contributors_questions(start=None, end=None, locale=None, product=None, count=10, page=1):
     """Get the top Support Forum contributors."""
 
     search = AnswerDocument.search()
 
-    search = search.filter(
-        # filter out answers by the question author
-        "script",
-        script="doc['creator_id'].value != doc['question_creator_id'].value",
-    ).filter(
-        # filter answers created between `start` and `end`, or within the last 90 days
-        "range",
-        created={"gte": start or datetime.now() - timedelta(days=90), "lte": end},
+    search = (
+        search.filter(
+            # filter out answers by the question author
+            "script",
+            script="doc['creator_id'].value != doc['question_creator_id'].value",
+        )
+        .filter(
+            # filter answers created between `start` and `end`, or within the last 90 days
+            "range",
+            created={"gte": start or datetime.now() - timedelta(days=90), "lte": end},
+        )
+        .extra(size=0)
     )
     if locale:
         search = search.filter("term", locale=locale)
     if product:
         search = search.filter("term", question_product_id=product.id)
 
+    # our filters above aren't perfect, and don't only return answers from contributors
+    # so we need to collect more buckets than `count`, so we can hopefully find `count`
+    # number of contributors within
     search.aggs.bucket(
-        # create buckets for the `count + 1` most active contributors
+        # create buckets for the `count * 10` most active users
```

Comment:
I mostly plucked it out of the air, but did a bit of playing around in Kibana to confirm that it would return almost all contributors over a 90 day period.